core/test/unit/models/session_spec.js

```diff
@@ -0,0 +1,54 @@
+const should = require('should');
+const models = require('../../../server/models');
+
+describe('Unit: models/session', function () {
```

Comment:
Was wondering about the missing upsert and destroy test. Let me know :)

Reply:
Oh nice - this logic was originally in another module (session store) and tested through that. will update :+1:

Reply:
Added tests for `permittedOptions`, `destroy` and `upsert` and a tiny test for the `user` relation setup thingy